packages/mobile/src/tokens/selectors.ts

```diff
@@ -60,11 +71,11 @@ export const tokensByCurrencySelector = createSelector(tokensListSelector, (toke
 })
 
 // Returns the token with the highest usd balance to use as default.
-export const defaultTokenSelector = createSelector(tokensListSelector, (tokens) => {
+export const defaultTokenSelector = createSelector(tokensWithTokenBalanceSelector, (tokens) => {
   let maxTokenAddress: string = ''
   let maxBalance: BigNumber = new BigNumber(-1)
   for (const token of tokens) {
-    const usdBalance = token.balance.multipliedBy(token.usdPrice)
+    const usdBalance = token.balance.multipliedBy(token.usdPrice ?? 0)
```

Comment:
If a token "doesn't have a USD price" is that equivalent to a USD price of 0?

Reply:
Hmm, not sure, I changed this to be based on `tokensWithUsdValueSelector` instead so we don't have to answer that :)

Reply:
It depends on how you think about "having a price of zero", but I don't think it should ever be zero. 

Not having a usd price happens when the token is not traded in any exchanges.

Having a price of zero should not ever happen.

I don't understand what you're aiming at though 🤔